packages/buttons/stories/6-IconButton.stories.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { IconButton } from '@zendeskgarden/react-buttons';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import ChevronDownIcon from '@zendeskgarden/svg-icons/src/16/chevron-down-stroke.svg';
+
+export default {
+  title: 'Components/Buttons/IconButton',
+  component: IconButton
+} as Meta;
+
+export const Default: Story = ({
+  isDanger,
+  size,
+  isPrimary,
+  isBasic,
+  isPill,
+  focusInset,
+  isRotated
+}) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <IconButton
```

Comment:
It would be good to show two or three different icons, just to demonstrate that they layout as expected. Also, I'd recommend not using the ChevronDownIcon svg here as it kind of duplicates the purpose of the `ChevronButton`.